README.md

```diff
@@ -12,18 +12,18 @@ I have not provided any test code, but you can base yours on test code from
 other exercises. Your tests must check the correctness of the result of running
 the function and run automatically when you commit through a GitHub action.
 
-https://stackoverflow.com/questions/49799475/generating-permutation-matrices-of-a-2-dimensional-array-in-python
+## Runtime Analysis
 
-https://www.geeksforgeeks.org/print-all-permutation-of-array-using-javascript/
+What is the worst-case big $\Theta$ time complexity of your algorithm?
 
-https://www.geeksforgeeks.org/transpose-a-two-dimensional-2d-array-in-javascript/
+There is always a n comparison, along with a $$2n^3$$ with a permutation matrix. It is then transposed which is $$n^2$$. All of this is done several times, with a value of n! for each possibility. This would make the final time complexity equal to $$O(n! * (n + n^2 + 2n^3))$$.
 
-I used the sources above to understand how the algorithm was supposed to work, as well as some help from ChatGPT to figure out what was needed to perfrom these actions.
+https://stackoverflow.com/questions/49799475/generating-permutation-matrices-of-a-2-dimensional-array-in-python
 
-## Runtime Analysis
+https://www.geeksforgeeks.org/print-all-permutation-of-array-using-javascript/
 
-What is the worst-case big $\Theta$ time complexity of your algorithm?
+https://www.geeksforgeeks.org/transpose-a-two-dimensional-2d-array-in-javascript/
 
-There is always a n comparison, along with a $$2n^3$$ with a permutation matrix. It is then transposed which is $$n^2$$. All of this is done several times, with a value of n! for each possibility. This would make the final time complexity equal to $$O(n! * (n + n^2 + 2n^3))$$.
+I used the sources above to understand how the algorithm was supposed to work, as well as some help from ChatGPT to figure out what was needed to perfrom these actions. I also based my test code on Powerfuljackell's test code.
```

Comment:
So what part of the code did you write?

Reply:
I wrote the code in code.js. The sources were used mainly to understand what the problem was asking.